dev-libs/inih/inih-56-r1.ebuild

```diff
@@ -1,26 +1,25 @@
 # Copyright 2020-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 # Tests not wired up to meson and don't seem to be intended for downstream use yet
 # e.g. hardcoding gcc, just a shell script
 
-inherit meson-multilib
+inherit meson
 
 DESCRIPTION="inih (INI not invented here) simple .INI file parser"
 HOMEPAGE="https://github.com/benhoyt/inih"
-
 SRC_URI="https://github.com/benhoyt/inih/archive/r${PV}.tar.gz -> ${P}.tar.gz"
-KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 sparc x86"
-S="${WORKDIR}/inih-r${PV}"
+S="${WORKDIR}"/inih-r${PV}
```

Comment:
That's a weird change.

Reply:
agreed

Reply:
You're the one who persuaded me to not overquote a while ago. Anyway, it's maintained by base-system@.